sdk/react/progressBar.js

```diff
@@ -13,27 +13,40 @@ import {
 
 var Utils = require('./utils');
 var styles = Utils.getStyles(require('./style/progressBarStyles.json'));
+var Log = require('./log');
 
 var ProgressBar = React.createClass({
   propTypes: {
-    percent: React.PropTypes.number
+    percent: React.PropTypes.number,
+    config: React.PropTypes.object,
+    ad: React.PropTypes.object
   },
 
   render: function() {
     var playedPercent = this.props.percent;
     var bufferedPercent = 0;
     var unbufferedPercent = 1 - playedPercent - bufferedPercent;
+
+    if (this.props.ad) {
+      var playedColor = this.props.config.controlBar.adScrubberBar.playedColor;
+      var backgroundColor = this.props.config.controlBar.adScrubberBar.backgroundColor;
+      var bufferedColor = this.props.config.controlBar.adScrubberBar.bufferedColor;
+    } else {
+      var playedColor = this.props.config.controlBar.scrubberBar.playedColor;
+      var backgroundColor = this.props.config.controlBar.scrubberBar.backgroundColor;
+      var bufferedColor = this.props.config.controlBar.scrubberBar.bufferedColor;
+    }
+
+    var playedStyle = {backgroundColor: playedColor, flex: playedPercent};
+    var backgroundStyle = {backgroundColor: backgroundColor, flex: bufferedPercent};
+    var bufferedStyle = {backgroundColor: bufferedColor, flex: unbufferedPercent};
     
-    var playedStyle = {backgroundColor: '#4389FF', flex: playedPercent};
-    var bufferedStyle = {backgroundColor: '#7F7F7F', flex: bufferedPercent};
-    var unbufferedStyle = {backgroundColor: '#AFAFAF', flex: unbufferedPercent};
-    
```

Comment:
I changed variable names to make sure they are aligned with the skin.json parameter names